### PR TITLE
rename title from "all the classic fors" to For loops

### PR DESCRIPTION
**Pull request by wilzbach**
+ bring structure to the game

### CODE REVIEW

File: public/content/en/basics.md [hunk 1]
```diff
@@ -745,30 +745,45 @@ void main() {
     writeln("My dstring: ", dstr);
 }
 
-# All the classic for's
+# For loops
```
Comment (wilzbach) [inline public/content/en/basics.md]: maybe also For/while loops?

Comment (stonemaster) [inline public/content/en/basics.md]: What about just "Loops"`

Comment (wilzbach) [inline public/content/en/basics.md]: The next section is called foreach loops

Comment (wilzbach) [inline public/content/en/basics.md]: Maybe "C-style loops"?

Comment (stonemaster) [inline public/content/en/basics.md]: The next section is called just foreach so "Loops" would be fine IMO. C-style loops is maybe awkward for someone coming from a non-C backward.

**Comment by wilzbach** [comment]
Rebased.

**Pull request merged**